spec/system/facility_list_tile_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Facility list tile", feature_setting: { facility_tile_list: true } do
+  # This spec requires that Facility have an attachement. Currently that can be done using
+  # either Paperclip or ActiveStorage. Because Paperclip requires a migration that may or
+  # may not have occurred, these specs are only set to run if ActiveStorage is being used.
+  if SettingsHelper.feature_on?(:active_storage)
+    Facility.include DownloadableFiles::Image
+
+    let(:admin_user) { create(:user, :administrator) }
+    let(:facility) { create(:facility, :with_image) }
+
+    it "shows image on home page" do
+      facility
+      visit facilities_path
+
+      expect(page).to have_selector(".tile-image")
+    end
+
+    context "image manpulation" do
+      before do
+        login_as admin_user
+        visit edit_facility_path(facility)
+      end
+
+      context "with image attached" do
```

Comment:
I'd suggest making with/without an attached image the top level context.  Fine to do this in a follow up PR